### ISSUE TITLE
Docker support

### ISSUE THREAD

**Issue by Cohee1207** [daswer123/rvc-python#23]
I guess it's the only thing left to achieve perfection. Imagine the steps being like these:

1. Add Dockerfiles for CPU and CUDA installs
2. Add GitHub Action to build images on release (could be both arm and x64 architecture)
3. Publish official Docker images on GHCR or Docker Hub

As a little proof of concept, I have prepared a working example of Dockerfile for CPU mode. See here: https://github.com/Cohee1207/rvc-python/commit/4f11cf4e2e883f231d39510552a54870937fa8fd

**Comment by daswer123** [comment]
Yeah, that's a good point. I'm working this week so I'll be able to do it closer to the weekend, if anything you can make any changes you want, I don't mind )
